backend/utils.js

```diff
@@ -252,7 +252,11 @@ const times = number => func => {
   return iter(0)
 }
 
-const possessive = string => (string.toLowerCase().endsWith('s') ? '&rsquo;' : '&rsquo;s')
+const possessive = string => {
+  if (!string) return ''
+
+  return `${string}${string.toLowerCase().endsWith('s') ? '’' : '’s'}`
+}
```

Comment:
ooh, does this do the right curly apostrophe in the view?

Reply:
Yeah it's the same as `&rsquo;`